books/kestrel/apt/isodata-tests.lisp

```diff
@@ -491,6 +491,110 @@
 
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 
+(must-succeed*
```

Comment:
These are great tests, but I would move them down within the file. Now they are between the tests for `oldp` and `newp`, which should probably stay one after the other, since they are related. Perhaps these could go just after "Wildcard in OLD." and "Check NEW-NAME input."?